superset/charts/api.py

```diff
@@ -706,6 +705,8 @@ def data(self) -> Response:
 
         try:
             command = ChartDataCommand()
+            # TODO: remove
+            json_body["viz_type"] = "cccs_grid"
```

Comment:
This shouldn't be merged. It's to test the rest of the code without [this](https://github.com/CybercentreCanada/superset-ui/pull/14/files) being able to be used until [CLDN-1045: Superset: Create azure pipeline for superset-ui repo](https://cccs.atlassian.net/browse/CLDN-1045) is done.

Reply:
Do you want us to hold off approving the PR until it is removed?

Reply:
I made a few additional changes here and there since a month ago (it's already been that long?) but otherwise I'm ready to merge this.